### PR TITLE
[sc-6615] release pipeline

### PR DESCRIPTION
**Pull request by steb-veksler**

### CODE REVIEW

File: .github/workflows/pr.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        with:
+          cache: true
+          go-version-file: go.mod
+          cache-dependency-path: go.sum
+      -
+        name: Install Dependencies
+        run: |
+          make generate_ebpf_in_docker
+          go mod download -x
+      # -
```
Comment (noamApps) [inline .github/workflows/pr.yaml:38]: ?

**Review by noamApps — CHANGES_REQUESTED** [review]
caretta.tgz in repo

File: .github/workflows/pr.yaml (hunk 1 above)
Comment (udiz) [reply .github/workflows/pr.yaml]: removed

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        with:
+          cache: true
+          go-version-file: go.mod
+          cache-dependency-path: go.sum
+      -
+        name: Install Dependencies
+        run: |
+          make generate_ebpf_in_docker
+          go mod download -x
+      # -
```
Comment (noamApps) [inline .github/workflows/release.yaml:40]: still here

**Review by noamApps — CHANGES_REQUESTED** [review]
change default grafana password to caretta
in the values
```
grafana:
  adminPassword: caretta
```

doc in README in #20

**Review by noamApps — APPROVED** [review]

**Pull request merged**